### PR TITLE
chore: Added Unity Metrics onboarding

### PR DESCRIPTION
**Pull request by bitsandfoxes**
Added onboarding for the frontend platform `Unity`.

### CODE REVIEW

File: static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx [hunk 1]
```diff
@@ -667,3 +667,75 @@ export const getDotnetMetricsOnboarding = ({
...
+
+const getUnityConfigureSnippet = () => `
+public override void Configure(SentryUnityOptions options)
+{
+    options.ExperimentalMetrics = new ExperimentalMetricsOptions
+    {
+      EnableCodeLocations = true
+    };
+}`;
+export const getUnityMetricsOnboarding = (): OnboardingConfig => ({
```
Comment (ArthurKnaus) [inline static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx:679]: Nit: as this is used in only one platform I would hardcode it directly in `unity.tsx` 🤷

**Review by ArthurKnaus — APPROVED** [review]
🚀

**Comment by smeubank** [comment]
@bitsandfoxes and issue to merge 🚀 ?

File: static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx (hunk 1 above)
Comment (smeubank) [reply static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx:679]: would be nice to apply more abstractions to all the platforms, but let's ,merge for now to unblock the users, and follow up on that later

Comment (ArthurKnaus) [reply static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx]: More abstraction potentially makes it harder to change the content and also makes us less flexible. Would be very careful with adding it here. We have to edit the docs content quite often. 🤔
But lets follow up on that 👍

**Pull request merged**